shared-data/js/helpers/getFlexStackerHardwareProps.ts

```diff
@@ -0,0 +1,77 @@
+import { FLEX_STACKER_MODULE_V1 } from '../constants'
+import { getModuleDef } from '../modules'
+import { getSchema2Dimensions } from './positionMath'
+
+import type { LabwareDefinition, ModuleModel, Vector3D } from '../types'
+
+export const getModuleMaxFillHeight = (model: ModuleModel): number => {
+  if (model === FLEX_STACKER_MODULE_V1) {
+    return (
+      getModuleDef(FLEX_STACKER_MODULE_V1).dimensions.maxStackerFillHeight ?? 0
+    )
+  }
+  throw new Error(`Invalid module model for max fill height: ${model}`)
```

Comment:
just a nit for protocol-designer, we prefer `console.errors` over `throw new Error` so you aren't left with a white screen. `console.error` will show up in sentry if the user hits it.
```suggestion
  console.error(`Invalid module model for max fill height: ${model}`)
```